backend/agents/data_store.py

```diff
@@ -0,0 +1,38 @@
+from .tool import toolMetadata
+from .types import Parameter
+from .agent import Agent, agentMetadata
+
+
+@toolMetadata(
+    name="Get Database Data",
+    description="This tool gets data from the database",

```

Comment:
Since this tool has a specific function (until we build the ability for generic queries), a more specific name and description may help the LLM to choose this function appropriately. "gets data" may suggest it can get any data from the database that may cause problems with certain requests.

Since transactions is now part of our ontology, should our description be something like "This tool gets the sum of all transactions for a specified merchant"?

I guess it depends on what types of database requests we expect this function to handle. Please ignore if this is intentionally generic for now.

Reply:
Yeah was keeping it deliberately generic for now. More than happy for this to be changed as the agent logic is implemented more